frontend/src/reducers/user.ts

```diff
@@ -1,32 +1,45 @@
 import { Routine } from 'redux-saga-routines';
-import { fetchUserRoutine } from '../routines/user';
+import { addNewUserRoutine, fetchUserRoutine } from '../routines/user';
 
-export interface IUserState {
+export interface IUserInitState {
```

Comment:
It should be in the global models and used for Store typing

Reply:
Maybe remove IUserInitState and use IUserState instead? And initialState will be { isLoading:  false, isAuthorized: false, data: null } of type IUserState